test/integration/reasoner/planner/PlannerTest.java

```diff
@@ -63,15 +68,37 @@ public class PlannerTest {
     private static ConceptManager conceptMgr;
     private static LogicManager logicMgr;
 
+    private static class ReasonerPlannerTestWrapper extends GreedyCostSearch.OldPlannerEmulator {
+
+        private final Set<Resolvable<?>> mockResolvables;
+        public ResolvableConjunction mockConjunction;
+
+        public ReasonerPlannerTestWrapper(CoreTransaction tx, Set<Resolvable<?>> mockResolvables) {
+            super(tx.traversal(), tx.concepts(), tx.logic());
+            this.mockResolvables = mockResolvables;
+            this.mockConjunction = ResolvableConjunction.of(new Conjunction(set(), new ArrayList<>()));
+        }
+
+        @Override
+        public Set<Resolvable<?>> compile(ResolvableConjunction conjunction) {
+            return mockResolvables;
+        }
+
+        private static List<Resolvable<?>> planResolvables(Set<Resolvable<?>> resolvables, Set<Identifier.Variable.Retrievable> inputBounds) {
+            ReasonerPlannerTestWrapper planner = new ReasonerPlannerTestWrapper(transaction, resolvables);
+            return planner.planConjunction(planner.mockConjunction, inputBounds).planOrder();
+        }
```

Comment:
It feels strange to create this extra class just to be able to create a new `planResolvables(Set<Resolvable>)` entry point. This tells me that we should have a `package-private` method in `ReasonerPlanner` (or `GreedyCostSearch)` that does exactly this.

Reply:
Ah this reminds me.
A key convenience of having the ReasonerPlanner do the compilation was that the compilation method was easy to mock like this, and didn't have to update every test in this file to actually construct a ResolvableConjunction and then select out the interesting concludables from there.